### ISSUE TITLE
The crop method '0' is not available for Imagick

### ISSUE THREAD

**Issue by endelwar** [oscarotero/imagecow#28]
Hi,
I'm getting `The crop method '0' is not available for Imagick` error when I pass `0` (int value zero) as third param of crop method like this:
```php
$imageStandard = Image::fromFile($imgSource . '/' . $filename)
                            ->autoRotate()
                            ->crop($cropData['width'], $cropData['height'], $cropData['offsetX'], $cropData['offsetY'])
                            ->resize($this->imageWidth, $this->imageHeight)
                            ->save($imgDestination . '/' . $filename);
```
$cropData is an array with int values (`['width' => 725, 'height' => 458, 'offsetX' => 0, 'offsetY' => 15]`)

On row 328 of Image.php `switch` statement is entered in any case and `getCropOffsets` method is run in any case (see this example on 3v4l https://3v4l.org/6eitl), even if `$x` is an integer.

A quick hack that "works for me"™ is wrapping switch statement in an `if (is_string($x)) {}`

**Comment by endelwar** [comment]
Well, I've forgot about php loose comparison (I've a good habit to use `===` and friends)!
Switch statement uses loose comparison so int value `0` is treated as false (or null): the if trick should be used in this case.

**Comment by oscarotero** [comment]
Hi, @endelwar 
This issue had been fixed in #27 but not tagged yet. I just released [v2.1.6](https://github.com/oscarotero/imagecow/releases/tag/v2.1.6).
Thank you

**Comment by endelwar** [comment]
Very good @oscarotero! I haven't see #27 before opening this new issue, sorry!
